apps/dao/src/components/PageAnalytics/CrvStats/index.tsx

```diff
@@ -0,0 +1,145 @@
+import { useEffect } from 'react'
+import styled from 'styled-components'
+
+import useStore from '@/store/useStore'
+import { formatNumber } from '@/ui/utils'
+import { t } from '@lingui/macro'
+
+import Box from '@/ui/Box'
+import MetricsComp, { MetricsColumnData } from '@/components/MetricsComp'
+import Tooltip from '@/ui/Tooltip'
+
+const CrvStats: React.FC = () => {
+  const provider = useStore((state) => state.wallet.getProvider(''))
+  const { veCrvData, getVeCrvData, veCrvFees, veCrvHolders } = useStore((state) => state.analytics)
+
+  const noProvider = !provider
+  const veCrvLoading = veCrvData.fetchStatus === 'LOADING'
+  const veCrvFeesLoading = veCrvFees.fetchStatus === 'LOADING'
+
+  useEffect(() => {
+    if (provider && veCrvData.totalCrv === 0 && veCrvData.fetchStatus !== 'ERROR') {
+      getVeCrvData(provider)
+    }
+  }, [veCrvData.totalCrv, veCrvData.fetchStatus, getVeCrvData, provider])
+
+  const veCrvApr =
+    veCrvLoading || veCrvFeesLoading ? 0 : (((veCrvFees.fees[1].fees_usd / veCrvData.totalVeCrv) * 52) / 0.3) * 100
```

Comment:
Doesn't look like the kind of calculation that should be in a component 

How about a hook or helper fn?